File: src/main/java/fr/marstech/mtlinkspray/repository/MtLinkSprayCollectionRepository.java
package fr.marstech.mtlinkspray.repository;

import fr.marstech.mtlinkspray.entity.MtLinkSprayCollectionItem;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

import java.util.Optional;

@Repository
public interface MtLinkSprayCollectionRepository extends MongoRepository<MtLinkSprayCollectionItem, String> {

    Optional<MtLinkSprayCollectionItem> findById(String id);
}

File: src/main/kotlin/fr/marstech/mtlinkspray/repository/MtLinkSprayCollectionRepository.kt
package fr.marstech.mtlinkspray.repository

import fr.marstech.mtlinkspray.entity.MtLinkSprayCollectionItem
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface MtLinkSprayCollectionRepository : MongoRepository<MtLinkSprayCollectionItem, String> {
    override fun findById(id: String): Optional<MtLinkSprayCollectionItem>
}
